lib/rubocop/cop/betterment/authorization_in_controller.rb

```diff
@@ -144,6 +144,19 @@ def contains_id_parameter?(params)
         def suspicious_id?(symbol_name)
           @unsafe_parameters.include?(symbol_name.to_sym) || @unsafe_regex.match(symbol_name) # symbol_name.to_s.end_with?("_id")
         end
+
+        def find_param_wrappers(class_methods)
+          class_methods.each_with_object([]) do |(method_name, method_returns), param_wrappers|
+            param_wrappers << method_name if method_returns.any? do |return_value|
+              name = Utils::Parser.get_root_token(return_value)
+              name.equal?(:params) || param_wrappers.include?(name)
+            end
+          end
+        end
```

Comment:
This is what we're calling a param wrapper,  right?

```ruby
def something_params
  params.require(:something).permit(...)
end
```

Reply:
Out of scope for this PR, but would this fail to recognize this as a param wrapper:
```ruby
def something_params
  @something_params ||= params.require(:something).permit(...)
end
```